scripts/deactivate_iac.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+
+import requests
+
+from app import config
+
+try:
+    config_info = getattr(config, os.environ['APP_SETTINGS'])
+except (AttributeError, KeyError) as e:
+    config_info = config.DevelopmentConfig
+
+# Put config into a dict
+config = dict((name, getattr(config_info, name)) for name in dir(config_info) if not name.startswith('__'))
+
+# Setup URLs
+case_url = f"{config['CASE_URL']}/cases/"
+iac_url = f"{config['IAC_URL']}/iacs/"
+ci_url = f"{config['COLLECTION_INSTRUMENT_URL']}/collection-instrument-api/1.0.2/collectioninstrument/id/"
+collex_url = f"{config['COLLECTION_EXERCISE_URL']}/collectionexercises/"
+sample_url = f"{config['SAMPLE_URL']}/samples/"
+eq_url = f"{config['EQ_URL']}"
+
+
+def main(collection_ex_id):
+    collection_ex_info = requests.get(collex_url + "link/" + collection_ex_id[0],
+                                      auth=config["COLLECTION_EXERCISE_AUTH"])
+    collection_ex_info.raise_for_status()
+
+    sample_summary_id = collection_ex_info.json()
+
+    sample_units = requests.get(sample_url + sample_summary_id[0] + "/sampleunits",
```

Comment:
Some simple print statements or logging for each (big) request might be useful, e.g.: "Fetching sample units" here.